.github/workflows/release.yml

```diff
@@ -0,0 +1,55 @@
+name: "Create Phar on release"
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  bundle:
+    name: "Create binary"
+    runs-on: "ubuntu-latest"
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Draft Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Release ${{ github.ref }}
+          draft: true
+          prerelease: false
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@master
+        with:
+          php-version: 8.0
```

Comment:
Will a phar created on PHP 8.0 run without issues on PHP 5.4 (which is the lowest supported PHP version) ?
This will need some investigation and it may be safer to create the phar with the lowest supported PHP version anyhow.

Reply:
Inversely, does building it on PHP 5.4 make it compatible with code in PHP 8.0 (like the `mixed` return type).

This warrants a test

Reply:
> Inversely, does building it on PHP 5.4 make it compatible with code in PHP 8.0 (like the mixed return type).

I believe it should, as I've made sure that the existing code is cross-version compatible with PHP 5.4 - 8.0. The phar wrapper only packages the code, doesn't it ? It doesn't _change_ it AFAIK.

Reply:
(testing this would be good though, quite possibly could even be included as a test in the CI, i.e. build the phar on PHP 5.4 in a separate job and make it available to the next job. Then in the next job, run the linting with both a direct call to the code as well as with the phar)

Reply:
I agree with jrfnl, build in lowest version and in release process check than phar do the same as base code.

Reply:
I changed the pipelines to compile on 5.4, then run test against all supported versions, and publish if those all pass.